apps/meteor/client/components/Sidebar/Header.tsx

```diff
@@ -16,7 +16,7 @@ const Header: FC<HeaderProps> = ({ title, onClose, children, ...props }) => {
 			{(title || onClose) && (
 				<Box display='flex' flexDirection='row' alignItems='center' pi='x24' justifyContent='space-between' flexGrow={1}>
 					{title && (
-						<Box fontSize='p2' fontWeight='p2' flexShrink={1} withTruncatedText>
```

Comment:
stroke-extra-dark, stroke is not a valid color, only border color  
<img width="399" alt="image" src="https://user-images.githubusercontent.com/5263975/219540283-1c459d99-0491-40f0-b408-2e4ff29d06da.png">